src/components/Favourites/favourites.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../styles/utils/text-styles.scss';
+
+.wrapper {
+  display: flex;
+  gap: 8px;
+  flex-direction: column;
+  align-items: start;
+  margin-bottom: 32px;
```

Comment:
instead of making wrapper and setting display flex, add margin-bottom to favourites__title